Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -862,6 +863,18 @@ invalidate_stats()
   self_intersect = false;
 }
 
+Scene_polyhedron_item::Vertex_selection_map 
+Scene_polyhedron_item::vertex_selection_map()
+{
+  return get(boost::vertex_index,*d->poly);
+}
+
+Scene_polyhedron_item::Face_selection_map 
+Scene_polyhedron_item::face_selection_map()
+{
+  return get(boost::face_index,*d->poly);
```

Comment:
There are two property tag `vertex_selection` and `face_selection`. Is not there a bug here?